### ISSUE TITLE
SQLSTATE [08P01]

### ISSUE THREAD

**Issue by pixari** [njbarrett/laravel-postgis#48]
Hello,

everytime I'm executing the "php artisan migrate" command, I get this error:

```
  [Illuminate\Database\QueryException]
  SQLSTATE[08P01]: \Unknown error>>: 7 ERROR:  bind message supplies 1 parameters, but prepared statement "pdo_stmt_00000003" requires 2 (SQL: select * from information_schema.tables where table_schema = migrations and table_name = ?)

  [PDOException]
  SQLSTATE[08P01]: \Unknown error>>: 7 ERROR:  bind message supplies 1 parameters, but prepared statement "pdo_stmt_00000003" requires 2
```

Do you know how to fix it?

Thanks.

**Comment by jubairsaidi** [comment]
Yea getting this issue as well.  the odd thing is that it seems like (somehow) phaza/laravel-postgis is overriding the internal Schema class of Laravel... this shouldn't happen..

**Comment by jubairsaidi** [comment]
https://github.com/Bosnadev/Database/commit/3f6c4ee5bfe3f1af39f97d7bf7a9f187e693b2c2 works

SO, in `phaza/laravel-postgis/src/Schema/Builder.php`
replacing `class Builder extends \Bosnadev\Database\Schema\Builder` with `class Builder extends \Illuminate\Database\Schema\PostgresBuilde` seems to fix this issue.  but I don't know what, if any, effects this has.

**Comment by mgldvlpr** [comment]
I also have a same problem with 

[Illuminate\Database\QueryException]
  SQLSTATE[08P01]: \Unknown error>>: 7 ERROR:  bind message supplies 1 parameters, but prepared statement "pdo_stmt_00000003" requires 2 (SQL: select * from information_schema.tables where table_schema = migrations and table_name = ?)

  [PDOException]
  SQLSTATE[08P01]: \Unknown error>>: 7 ERROR:  bind message supplies 1 parameters, but prepared statement "pdo_stmt_00000003" requires 2

**Comment by jonnybarnes** [comment]
Hopefully my PR should fix this: pull#49
